app/marketing/views.py

```diff
@@ -615,7 +618,7 @@ def leaderboard(request, key=''):
     else:
         amount_max = 0
 
-    is_linked_to_profile = '_tokens' in key or '_keywords' in key
+    is_linked_to_profile = '_tokens' in key or '_keywords' in key or '_cities' in key or '_countries' in key or '_continents' in key
```

Comment:
If we think this might continue to grow, we could also use something like:

```python
profile_keys = ['_tokens', '_keywords', '_cities', '_countries', '_continents']
is_linked_to_profile = any(sub in key for sub in profile_keys)
```